package.json

```diff
@@ -142,7 +142,7 @@
     "dedent": "^0.7.0",
     "del": "3.0.0",
     "detect-port": "^1.3.0",
-    "electron": "18.0.4",
+    "electron": "18.3.0",
```

Comment:
A regression was introduced in Electron 18 that broke compatibility with Linux Arm, but it's fixed.